src/octokit.ts

```diff
@@ -2,12 +2,13 @@ import { Octokit } from "@octokit/rest";
 import { retry } from "@octokit/plugin-retry";
 import program from "./parser/command-line";
 import configuration from "./configuration/config-reader";
+import { paginateGraphQL, paginateGraphQLInterface } from "@octokit/plugin-paginate-graphql";
 
-const customOctokit = Octokit.plugin(retry);
+const customOctokit = Octokit.plugin(retry, paginateGraphQL);
 
-let octokitInstance: Octokit | null = null;
+let octokitInstance: (Octokit & paginateGraphQLInterface) | null = null;
```

Comment:
maybe it's possible to use `typeof customOctokit`?

Reply:
I tried before, and the `paginate` from GraphQl Paginate doesn't seem to be deduced properly. Am I doing something wrong?

![image](https://github.com/user-attachments/assets/a3af669b-d687-4cab-9f7b-12f1e9822dc5)

Reply:
try to wrap in `InstanceType<>` because I think the type from constructor is different than instance type

Reply:
Oh nice one. Will push it in https://github.com/ubiquibot/conversation-rewards/pull/104 thank you.